docs/examples/use_cases/paddle/resnet50/paddle-resnet50.rst

```diff
@@ -1,49 +1,48 @@
 ResNet Training in PaddlePaddle
 ===============================
 
-This simple demo showcases ResNet50 training on ImageNet.
+This is a demo showcasing ResNet50 training on ImageNet.
+The code is based on `NVIDIA Deep Learning Examples <https://github.com/NVIDIA/DeepLearningExamples/tree/master/PaddlePaddle/Classification/RN50v1.5>`_
 
-Run it with the commands below:
+Data augmentation
+-----------------
 
-.. code-block:: bash
-
-   python -m paddle.distributed.launch --selected_gpus 0,1,2,3,4,5,6,7 train.py -b 128 -j 4 [imagenet-folder with train and val folders]
+This model uses the following data augmentation:
 
-Training
---------
+* For training:
+  * Normalization
+  * Random resized crop to 224x224
+    * Scale from 8% to 100%
+    * Aspect ratio from 3/4 to 4/3
+  * Random horizontal flip
+* For inference:
+  * Normalization
+  * Scale to 256x256
+  * Center crop to 224x224
```

Comment:
It looks like the middle tier of the list is not parsed as such.

![image](https://github.com/NVIDIA/DALI/assets/11878086/5437ace2-6210-49fe-98a3-d9b827fea6e5)

Reply:
Maybe now